CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.20)
+
+project(qrack_device)
+
+set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+add_library(qrack_device SHARED pennylane_qrack/qrack_device.cpp)
+target_include_directories(qrack_device PUBLIC /home/iamu/GitHub/catalyst/runtime/include)
```

Comment:
This path seems contingent on a local machine path.

Reply:
Fixed. (We use Catalyst as a submodule now, to get its headers.)